### DISCUSSION TITLE
probability matching mean for ensemble precipitation/reflectivity with MET/METplus

### DISCUSSION THREAD

**Discussion by JiliDong-NOAA** [dtcenter/METplus#1339] [Statistical Computation] [answered]
Hi,

I am wondering if METplus has the capability of calculating the probability matching mean or localized probability matching mean for precipitation/reflectivity of an ensemble forecast? If yes, where could I find the related document/guidance?

Thanks!

Jili

**Comment by jprestop** [comment, marked as answer]
Hi @JiliDong-NOAA.  Thank you for your question.  This capability has been outside the scope of METplus/MET as the focus of Ensemble-Stat was to be a tool for verifying ensemble data, and the simplistic ensemble post-processing it did was secondary.   However, we understand that the need exists for common ensemble post processing tools in the tools.  While we are not opposed to enhancing the new Gen-Ens-Prod tool to support probability-matched-mean and other techniques that are in common use and reasonable to implement, but that would require interest and funding from our sponsors.

For now, I'll tag @LoganDawson-NOAA to see if he knows if the SRW (Short Range Weather) application is using it and how they produce it.

**Reply by LoganDawson-NOAA** [reply]
Hi @JiliDong-NOAA, Julie is correct that the probability-matched mean products are beyond the scope of METplus. Currently, the ensemble product generation for the SRW application is effectively retained within the operational code for HREF. I believe Matt Pyle has adapted the HREF ensemble product generation code to create the PMM and LPMM products you're looking for. I'll send you an email about it, as it probably isn't of relevance to other METplus users.

@jprestop I'm not sure what the long-term plan is for the ensemble post-processing tools in the UFS, but if there's ever the need or desire to incorporate some of these capabilities into the Gen-Ens-Prod tool, I'm sure we could have a discussion about how to make that happen.

**Reply by jprestop** [reply]
Thanks so much @LoganDawson-NOAA!  We appreciate the offer for future discussion on this topic.

@JiliDong-NOAA, I'd like to mark this discussion as being answered and "lock it" to prevent future posts. That's how we encourage users to create new discussions for new questions. But I wanted to give you an opportunity to comment on it before doing so. Please feel free to select a response as an answer to your question. That'll help future users with similar questions find answers.

**Reply by JiliDong-NOAA** [reply]
Thanks Julie and Logan! I have got the products from Matt as Logan noted. Appreciate all your replies and help.

Jili

**Reply by jprestop** [reply]
Thanks, @JiliDong-NOAA.  At this point I'm going to LOCK this conversation to prevent future posts. Our team has decided that once discussions have been answered, and the answer has been confirmed, we'll lock them. We want to encourage users to ask new questions in new discussions rather than posting to old ones. Hopefully that'll make the questions/answers easier for other users to follow.  So, if/when more issues/questions arise, please feel free to start a new discussion.
